etc/submit_py3.sh

```diff
@@ -211,21 +211,27 @@ $pythonCommand Startup.py
 jobrc=$?
 echo -e "======== WMAgent Run the job FINISH at $(TZ=GMT date) ========\n"
 
+echo "WMAgent bootstrap: WMAgent finished the job, it's copying the pickled report"
+set -x
+cp WMTaskSpace/Report*.pkl ../
+ls -l WMTaskSpace
+ls -l WMTaskSpace/*
+
 echo "======== WMAgent add timestamps metrics to FJR $outputFile ========"
+cd ../
 timeEndSec=`date +%s`
-reportIn=$outputFile
-$pythonCommand Timestamps.py --reportFile=$outputFile --wmJobEnd=$timeEndSec --wmJobStart=$timeStartSec
+echo "WMAgent adding WMTiming metrics"
+tpy=/srv/job/Utils/Timestamps.py
```

Comment:
Valentin, I just asked this question in the SI mattermost channel and instead of hard-wiring `/srv` here, the suggestion is to use `$HOME`, which defaults to `/srv` in the singularity images.